__tests__/components/editor/PropertyRemark.test.js

```diff
@@ -22,8 +26,11 @@ describe('<PropertyRemark />', () => {
   })
 
   it('displays just a string label if remark is not a valid URL', () => {
-    const no_link_wrapper = shallow(<PropertyRemark remark="A test remark"
-        label="Example RDA" />)
+    const no_link_wrapper = mount(
+      <PropertyRemark remark="A test remark" label="Example RDA" >
+        <OverlayTrigger overlay={<div />} />
```

Comment:
Can you say more about the changes made to this file? What's the role of the `OverlayTrigger` in this work?

Reply:
OK, I think I get it now. It's used to position the popover?

Reply:
The `OverlayTrigger` wraps a node that is a trigger for some kind of react-bootstrap event (like a `Popover`). I don't think it so much positions the popover as triggers it from (in this case) the span that positions it.

https://react-bootstrap.github.io/components/overlays/#popovers

Reply:
Removed the need for a mount with `OverlayTrigger` from this test